### PR TITLE
Add attr ‘isDoubleClick’, double click to clear pad

### PR DESCRIPTION
**Pull request by xiaoyaolml**
Add attr ‘clearOnDoubleClick’,  this feature is disabled by default

### CODE REVIEW

**Comment by gcacace** [comment]
Please rebase as you have conflicts with the current branch.

File: signature-pad/src/main/res/values/attrs.xml [hunk 1]
```diff
@@ -5,5 +5,6 @@
         <attr name="maxWidth" format="dimension" />
         <attr name="penColor" format="color" />
         <attr name="velocityFilterWeight" format="float" />
+        <attr name="isDoubleClick" format="boolean"/>
```
Comment (gcacace) [inline signature-pad/src/main/res/values/attrs.xml]: Please use a meaningful attribute name, like `clearOnDubleClick`.

File: signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java [hunk 2]
```diff
@@ -60,6 +65,7 @@ public SignaturePad(Context context, AttributeSet attrs) {
             mMinWidth = a.getDimensionPixelSize(R.styleable.SignaturePad_minWidth, convertDpToPx(3));
             mMaxWidth = a.getDimensionPixelSize(R.styleable.SignaturePad_maxWidth, convertDpToPx(7));
             mVelocityFilterWeight = a.getFloat(R.styleable.SignaturePad_velocityFilterWeight, 0.9f);
+            mIsDoubleClick = a.getBoolean(R.styleable.SignaturePad_isDoubleClick, true);
```
Comment (gcacace) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: I would prefer this feature to be disabled by default.

File: signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java [hunk 3]
```diff
@@ -156,6 +162,22 @@ public boolean onTouchEvent(MotionEvent event) {
             case MotionEvent.ACTION_DOWN:
                 getParent().requestDisallowInterceptTouchEvent(true);
                 mPoints.clear();
+
+                if (mIsDoubleClick) {
+                    if (firstClick != 0 && System.currentTimeMillis() - firstClick > 200) {
```
Comment (gcacace) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: Could you move this logic in a private function?

File: signature-pad/src/main/res/values/attrs.xml (hunk 1 above)
Comment (xiaoyaolml) [inline signature-pad/src/main/res/values/attrs.xml]: get it

File: signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java [hunk 4]
```diff
@@ -45,8 +45,14 @@
...
     private float mVelocityFilterWeight;
     private OnSignedListener mOnSignedListener;
+    private boolean mClearOnDoubleClick;
+
+    //Click values
+    private long firstClick;
+    private int countClick;
 
     private Paint mPaint = new Paint();
+    private Path mPath = new Path();
```
Comment (gcacace) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: Path has been removed recently.

File: signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java [hunk 5]
```diff
@@ -45,8 +45,14 @@
     private int mMaxWidth;
     private float mVelocityFilterWeight;
     private OnSignedListener mOnSignedListener;
+    private boolean mClearOnDoubleClick;
+
+    //Click values
+    private long firstClick;
```
Comment (gcacace) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: Please respect the naming convention for member variables.

Comment (xiaoyaolml) [inline signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java]: Oh,sorry.

**Pull request closed without merging**